<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LPKGReader</class>
 <widget class="QWidget" name="LPKGReader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="filelistWidget"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="readPackage">
         <property name="text">
          <string>Read Package</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="generateFile">
         <property name="text">
          <string>Unpack files</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>filelistWidget</zorder>
  <zorder>readPackage</zorder>
  <zorder>generateFile</zorder>
  <zorder>generateFile</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
